packages/fxa-settings/src/components/Settings/ProductPromo/index.test.tsx

```diff
@@ -52,4 +58,28 @@ describe('ProductPromo', () => {
       MonitorLink
     );
   });
+  it('renders Monitor Plus promo if user does not have Monitor Plus', async () => {
+    const account = {
+      attachedClients: [
+        {
+          name: MozServices.Monitor,
+        },
+      ],
+      subscriptions: [],
+    } as unknown as Account;
+    renderWithLocalizationProvider(
+      <AppContext.Provider value={mockAppContext({ account })}>
+        <ProductPromo />
+      </AppContext.Provider>
+    );
+
+    screen.getByAltText('Mozilla Monitor');
+    screen.getByText(
+      'Privacy Matters: Find where your private info is exposed and take it back'
+    );
+    expect(screen.getByRole('link', { name: /Get started/ })).toHaveAttribute(
+      'href',
+      MonitorPlusLink
+    );
```

Comment:
👍 Can we get a Storybook state for this? Makes it easier to review and Eduardo to look at later too.